tests/typecheck/managers/querysets/test_as_manager.yml

```
@@ -212,3 +212,21 @@
                 class MyModel(models.Model):
                     from_queryset = FromQuerySet()
                     as_manager = ModelQuerySet.as_manager()
+
+-   case: nested_queryset_class_definition
+    main: |
+        from myapp.models import MyModel
+        reveal_type(MyModel.objects)  # N: Revealed type is "myapp.models.ManagerFromMyQuerySet[myapp.models.MyModel]"
+    installed_apps:
+        - myapp
+    files:
+        -   path: myapp/__init__.py
+        -   path: myapp/models.py
+            content: |
+                from django.db import models
+
+                class MyModel(models.Model):
+                    class MyQuerySet(models.QuerySet):
```

Not really related to this PR, but I'm finding it odd that the plugin dishonors/repairs `disallow_any_generics`.

To be correct under that config, it should look like below here

```suggestion
                    class MyQuerySet(models.QuerySet["MyModel"]):
```

@intgr I know removing the auto population of missing generics would be quite breaking. But do you think it's worth entertaining the idea?

(I think the case is similar for the manager test in this change set)

---

I think the *real* reason is that we don't set `disallow_any_generics` in django-stubs and it's [disabled by default](https://mypy.readthedocs.io/en/stable/config_file.html#confval-disallow_any_generics) in mypy. 😆

---

But, I think, until Django supports all `__class_getitem__` our monkey patching expects. It's impossible to have a successful run in or with `django-stubs` under `disallow_any_generics` (?)